### PR TITLE
Add highlights for 2021-06-18

### PR DESCRIPTION
**Pull request by marians**

### CODE REVIEW

File: src/content/changes/highlights/2021-06-11.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+## General
+
+As we aim to align our terminology with the Kubernetes project where possible, we started replacing the term _master node_ with _control plane node_.
+
+## Managed apps
+
+[Loki v0.2.0](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.2.0/) comes with upstream version 2.2.1. Please note that there is a manual intervention required when upgrading to this version. Check the release info for details.
+
```
Comment (marians) [inline src/content/changes/highlights/2021-06-11.md:17]: @ubergesundheit Can you help me say something interesting/valuable about that release?

File: src/content/changes/highlights/2021-06-11.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+As we aim to align our terminology with the Kubernetes project where possible, we started replacing the term _master node_ with _control plane node_.
+
+## Managed apps
+
+[Loki v0.2.0](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.2.0/) comes with upstream version 2.2.1. Please note that there is a manual intervention required when upgrading to this version. Check the release info for details.
```
Comment (cokiengchiara) [inline src/content/changes/highlights/2021-06-11.md:16]:
```suggestion
[Loki v0.2.0](https://docs.giantswarm.io/changes/managed-apps/loki-app/v0.2.0/) comes with upstream version 2.2.1. You can now use an existing secret (such as sealed secrets) for configuration, making the app more flexible. Please note that there is a manual intervention required when upgrading to this version. Check the release info for details.
```

Comment (cokiengchiara) [reply src/content/changes/highlights/2021-06-11.md:16]: @ubergesundheit I suggested something. Does it make sense?

(All I really know is this was a blocker for a customer to use our Loki set up)

Comment (ubergesundheit) [reply src/content/changes/highlights/2021-06-11.md]: Perfect! <3

**Comment by marians** [comment]
Thanks @cokiengchiara ! Are we done from your side?

**Comment by cokiengchiara** [comment]
@marians I added app that were recently upgraded. The main value of these upgrades is staying close to upstream (30 day upgrade promise), and not any specific value provided by any specific release.

@giantswarm/team-halo if you have anything to add on the upgrades, feel free to do so. Otherwise, for me, this is good to go.

**Review by webwurst — APPROVED** [review]

**Pull request merged**